docs/strategy.md

```diff
@@ -0,0 +1,13 @@
+# Strategy upload 
+User will upload strategies, and we will store the strategies under strategies/\<username> folder and run a pylint on the strategy. If it passes the pylint check, the folder will be saved and notify users, otherwise the folder will be deleted and users will be notified.
+
+strategies will be store under strategy+ \<id> folder under a username folder. The strategy will also name "main.py"
+
+# database storage
+we stick to the schema in strategies table under alchemist.db
+## location
+the location is relative to the flask app, e.g. strategies/Michael/strategy7/main.py
```

Comment:
based on your answer here, could you then please add a sample strategy path? that will help everyone understand what the strategy folder structure looks like. 🙏

Reply:
Sure, I will do it and push again